README.md

```diff
@@ -41,20 +41,18 @@ package main
 import "github.com/pusher/pusher-http-go"
 
 func main(){
+    // instantiate a client
+    client := pusher.Client{
+        AppId: "your_app_id",
+        Key: "your_app_key",
+        Secret: "your_app_secret",
+        Cluster: "your_app_cluster",
+    }
 
-	// instantiate a client
-	client := pusher.Client{
-	  AppId: "your_app_id",
-	  Key: "your_app_key",
-	  Secret: "your_app_secret",
-	  Cluster: "your_app_cluster",
-	}
-
-	data := map[string]string{"message": "hello world"}
-
-	// trigger an event on a channel, along with a data payload
-	client.Trigger("test_channel", "my_event", data)
+    data := map[string]string{"message": "hello world"}
 
+    // trigger an event on a channel, along with a data payload
+    client.Trigger("test_channel", "my_event", data)
```

Comment:
Maybe change this to my_channel - this is the one the debug console subscribes to

Reply:
It's actually `my-channel`, but yes I agree that is a worthwhile improvement. I'll make the change!